src/cli/mod.rs

```diff
@@ -56,36 +59,36 @@ pub(crate) struct CargoArgs {
 #[clap(author, version, about)]
 pub(crate) struct Args {
     /// Path of the project folder
-    #[clap(long = "path_file", value_hint = clap::ValueHint::DirPath)]
-    pub(crate) path_file: PathBuf,
+    #[clap(long, required = true, value_hint = clap::ValueHint::DirPath)]
+    pub(crate) project_path: PathBuf,
+    /// Path of the grcov json file with coveralls format
+    #[clap(long, required = true, conflicts_with = "covdir", value_parser = GrcovFormat::<&str>::coveralls_parser, value_hint = clap::ValueHint::DirPath)]
+    coveralls: Option<GrcovFormat<PathBuf>>,
+    /// Path of the grcov json file with covdir format
+    #[clap(long, required = true, conflicts_with = "coveralls", value_parser = GrcovFormat::<&str>::covdir_parser, value_hint = clap::ValueHint::DirPath)]
+    covdir: Option<GrcovFormat<PathBuf>>,
     /// Choose complexity metric to use along with thresholds values
-    #[arg(short, value_parser = parse_key_val::<Complexity, Thresholds>, long_help = thresholds_long_help())]
+    #[clap(long, short = 'c', default_value = "cyclomatic:50.0,0.7,65.0,45.0", value_parser = parse_key_val::<Complexity, Thresholds>, long_help = thresholds_long_help())]
```

Comment:
We can use `default_value_t` instead of `default_value`

Reply:
I left `default_value` for the `json` argument, because it is of type `PathBuf` so it is not possible to implement for it the `Display` trait which is mandatory to be able to use `default_value_t`.